src/avfoundation.cs

```diff
@@ -13552,6 +13847,36 @@ interface AVContentKeyResponse {
 		AVContentKeyResponse _InitWithAuthorizationToken (NSData authorizationTokenData);
 	}
 
+	[TV (14,5), Mac (11,3), iOS (14,5), Watch (7,4)]
+	[BaseType (typeof(NSObject))]
+	interface AVContentKeySpecifier
+	{
+		// TODO https://github.com/xamarin/xamarin-macios/issues/10904
+		[Static]
+		[Export ("contentKeySpecifierForKeySystem:identifier:options:")]
+		AVContentKeySpecifier GetContentKeySpecifier (AVContentKeySystem keySystem, NSObject contentKeyIdentifier, NSDictionary<NSString, NSObject> options);
+
+		[Export ("initForKeySystem:identifier:options:")]
+		IntPtr Constructor (AVContentKeySystem keySystem, NSObject contentKeyIdentifier, NSDictionary<NSString, NSObject> options);
+
+		[Export ("keySystem")]
+		AVContentKeySystem KeySystem { get; }
+
+		[Export ("identifier")]
+		NSObject Identifier { get; }
+
+		[Export ("options")]
+		NSDictionary<NSString, NSObject> Options { get; }
+	}
+
+	[TV (14,5), Mac (11,3), iOS (14,5), Watch (7,4)]
+	[BaseType (typeof(NSObject))]
+	interface AVContentKey
```

Comment:
see note where this `using AVContentKey = System.IntPtr;` was used for manual bindings